src/app/init.ts

```diff
@@ -0,0 +1,22 @@
+import { app } from 'electron';
+import * as path from 'path';
+
+import { isDevEnv } from '../common/env';
+import { getCommandLineArgs } from '../common/utils';
+
+// Handle custom user data path from process.argv
+const userDataPathArg: string | null = getCommandLineArgs(process.argv, '--userDataPath=', false);
```

Comment:
Can we not just have this file's entire logic in a method in `main.ts`

Reply:
No, because there are other `import` statements that get initiated